### PR TITLE
Sync protobuf version in every CI: Install protobuf 3.11.3 in LinuxRelease CI

### PR DESCRIPTION
**Pull request by jcwchen**
**Description**
Previously apt-get version of protobuf should be an old version... To sync protobuf version in every CI, install protobuf 3.11.3 in LinuxRelease CI

**Motivation and Context**
Sync protobuf version in every CI

### CODE REVIEW

File: .github/workflows/release_linux_x86_64.yml [hunk 1]
```diff
@@ -34,11 +34,24 @@ jobs:
         python-version: ${{ matrix.python-version }}
         architecture: ${{ matrix.architecture }}
 
+    - name: Install protobuf
```
Comment (askhade) [inline .github/workflows/release_linux_x86_64.yml:37]: why are you installing protobuf here?
entrypoint.sh does this right?
https://github.com/onnx/onnx/blob/master/.github/workflows/manylinux/entrypoint.sh#L18

Comment (jcwchen) [reply .github/workflows/release_linux_x86_64.yml:37]: entrypoint.sh installs protobuf inside the docker for building the wheel. The protobuf installation here is in the GitHub Action environment for testing the wheel.

Comment (askhade) [reply .github/workflows/release_linux_x86_64.yml:37]: can you add this comment to the yml to make this more clear. Thanks!

Comment (askhade) [reply .github/workflows/release_linux_x86_64.yml:37]: also since this is relevant for testing can we move this below just before testing starts?

Comment (jcwchen) [reply .github/workflows/release_linux_x86_64.yml]: Updated. Thanks for the suggestions!

**Review by askhade — APPROVED** [review]

**Pull request merged**